extensions/navigation/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+    "compilerOptions": {
+        "declaration": true,
+        "noResolve": false,
+        "jsx": "react",
+        "module": "commonjs",
+        "target": "es5",
+        "experimentalDecorators": true,
+        "noImplicitAny": true,
+        "outDir": "dist/",
+        "lib": [
```

Comment:
I fixed it for react-native. For the web, it is way more tricky, and I don't have enough context for it now.